src/main/features/claimant-response/routes/check-and-send.ts

```diff
@@ -12,6 +12,80 @@ import { DraftService } from 'services/draftService'
+import { FormaliseRepaymentPlanOption } from 'claimant-response/form/models/formaliseRepaymentPlanOption'
+import { CCJClient } from 'claims/ccjClient'
+import { PartyType } from 'common/partyType'
+import { Individual } from 'claims/models/details/theirs/individual'
+import { YesNoOption } from 'models/yesNoOption'
+import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
+import { MomentFactory } from 'shared/momentFactory'
+import { Moment } from 'moment'
+import { FullAdmissionResponse } from 'claims/models/response/fullAdmissionResponse'
+import { PartialAdmissionResponse } from 'claims/models/response/partialAdmissionResponse'
+import { RepaymentPlan } from 'claims/models/response/core/repaymentPlan'
+import { RepaymentPlan as CCJRepaymentPlan } from 'claims/models/replaymentPlan'
+import { PaymentIntention } from 'shared/components/payment-intention/model/paymentIntention'
+import { PaymentOption } from 'main/common/components/payment-intention/model/paymentOption'
+import { LocalDate } from 'forms/models/localDate'
+import { PaymentSchedule } from 'ccj/form/models/paymentSchedule'
+import { PaymentDate } from 'main/common/components/payment-intention/model/paymentDate'
+import { PaymentType } from 'shared/components/payment-intention/model/paymentOption'
+import { PaymentPlan } from 'main/common/components/payment-intention/model/paymentPlan'
+
+function getDateOfBirth (claim: Claim): Moment {
+  const defendant = claim.response.defendant
+  if (defendant.type === PartyType.INDIVIDUAL.value) {
+    const user: Individual = new Individual().deserialize(defendant)
+    return MomentFactory.parse(user.dateOfBirth)
+  }
+  return undefined
+}
+
+function getDefendantPaymentIntention (claim: Claim): PaymentIntention {
+  const response: FullAdmissionResponse | PartialAdmissionResponse = claim.response as FullAdmissionResponse | PartialAdmissionResponse
+  const paymentIntention = new PaymentIntention()
+
+  if (response.paymentIntention) {
+    paymentIntention.paymentOption = new PaymentOption(PaymentType.valueOf(response.paymentIntention.paymentOption))
+    paymentIntention.paymentDate = response.paymentIntention.paymentDate && new PaymentDate(
+      new LocalDate(response.paymentIntention.paymentDate.year(), response.paymentIntention.paymentDate.month(), response.paymentIntention.paymentDate.date())
+    )
+
+    const repaymentPlan: RepaymentPlan = response.paymentIntention.repaymentPlan
+
+    paymentIntention.paymentPlan = repaymentPlan && new PaymentPlan(claim.totalAmountTillToday,
+      repaymentPlan.instalmentAmount,
+      new LocalDate(repaymentPlan.firstPaymentDate.year(), repaymentPlan.firstPaymentDate.month(), repaymentPlan.firstPaymentDate.date()),
+      PaymentSchedule.of(repaymentPlan.paymentSchedule)
+    )
+  }
+  return paymentIntention
+}
+
+function countyCourtJudgement (claim: Claim, draft: Draft<DraftClaimantResponse>): CountyCourtJudgment {
```

Comment:
This should be in separate file, the same way as we have done `src/main/app/claims/ccjModelConverter.ts`. Then it should be unit tested.